extensions/wikia/JSVariables/JSVariables.php

```diff
@@ -58,6 +58,10 @@ function wfJSVariablesTopScripts(Array &$vars, &$scripts) {
 		 * End remove
 		 */
 	}
+	$vars['wikiaIsPowerUserAdmin'] = $user->isSpecificPowerUser( \Wikia\PowerUser\PowerUser::TYPE_ADMIN );
+	$vars['wikiaIsPowerUserFrequent'] = $user->isSpecificPowerUser( \Wikia\PowerUser\PowerUser::TYPE_FREQUENT );
+	$vars['wikiaIsPowerUserLifetime'] = $user->isSpecificPowerUser( \Wikia\PowerUser\PowerUser::TYPE_LIFETIME );
+
```

Comment:
I assume that these variables will be set to `false` in most cases. If so, please only set them when they're true - we already have 350+ global JS variables. No reason to add more than needed ;)